CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Versions
 ========
 
+2017.01.24
+----------
+
+* Add a new Golang image
```

Comment:
The tags should be normalized in the Changelog file, there is a mix of `.` & `-`, should be identical to git tags (`.`).